pywbem/cim_operations.py

```diff
@@ -5557,22 +5563,25 @@ class (or subclasses).
 
           IncludeQualifiers (:class:`py:bool`):
             Indicates that qualifiers are to be included in the returned
-            instances (or classes), as follows:
+            instances, as follows:
 
             * If `False`, qualifiers are not included.
-            * If `True`, qualifiers are included if the WBEM server implements
-              support for this parameter.
+            * If `True`, qualifiers are included if this method uses
+              traditional operations and the WBEM server implements support for
+              this parameter.
             * If `None`, this parameter is not passed to the WBEM server, and
               causes the server-implemented default to be used. :term:`DSP0200`
               defines that the server-implemented default is `False`.
 
-            This parameter has been deprecated in :term:`DSP0200`. Clients
-            cannot rely on qualifiers to be returned in this operation.
+            If this method uses pull operations, this parameter will be ignored
+            and the returned instances will not contain any qualifiers.
+            If this method uses traditional operations, clients cannot rely
+            on returned instances containing qualifiers, as described in
+            :term:`DSP0200`.
```

Comment:
I notice that you removed the statement about deprecation. "This parameter has been deprecated in :term:`DSP0200`. Shouldn't we just leave that in